Sources/Testing/ExitTests/ExitTest.Condition.swift

```diff
@@ -0,0 +1,148 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024–2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+private import _TestingInternals
+
+@_spi(Experimental)
+#if SWT_NO_EXIT_TESTS
+@available(*, unavailable, message: "Exit tests are not available on this platform.")
+#endif
+extension ExitTest {
+  /// The possible conditions under which an exit test will complete.
+  ///
+  /// Values of this type are used to describe the conditions under which an
+  /// exit test is expected to pass or fail by passing them to
+  /// ``expect(exitsWith:observing:_:sourceLocation:performing:)`` or
+  /// ``require(exitsWith:observing:_:sourceLocation:performing:)``.
+  public struct Condition: Sendable {
+    /// An enumeration describing the possible requirements for an exit test.
+    private enum _Kind: Sendable, Equatable {
+      /// The exit test must exit with a particular exit status.
+      case statusAtExit(StatusAtExit)
+
+      /// The exit test must exit with any failure.
+      case failure
+    }
+
+    /// The kind of requirement.
+    private var _kind: _Kind
+  }
+}
+
+// MARK: -
+
+@_spi(Experimental)
+#if SWT_NO_EXIT_TESTS
+@available(*, unavailable, message: "Exit tests are not available on this platform.")
+#endif
+extension ExitTest.Condition {
+  /// The process terminated successfully with exit code `EXIT_SUCCESS`.
```

Comment:
The phrasing of this documentation feels weird for a property. (Similar comment for `.failure`, `.exitCode(_:)`, and `.signal(_:)`.) Maybe something like this?
```suggestion
  /// A condition which matches when a process terminates successfully
  /// with exit code `EXIT_SUCCESS`.
```

Reply:
Just a typo from an earlier revision that I didn't post. Will fix.